src/EnergyPlus/WeatherManager.cc

```diff
@@ -2334,8 +2338,13 @@ namespace WeatherManager {
         state.dataEnvrn->OpaqueCloudCover = state.dataWeatherManager->TodayOpaqueSkyCover(state.dataGlobal->TimeStep, state.dataGlobal->HourOfDay);
 
         if (state.dataWeatherManager->UseRainValues) {
-            state.dataEnvrn->IsRain = state.dataWeatherManager->TodayIsRain(
-                state.dataGlobal->TimeStep, state.dataGlobal->HourOfDay); //.or. LiquidPrecipitation >= .8d0)  ! > .8 mm
+            // following the dev branch lower bound condition
+            if (state.dataWeatherManager->NumIntervalsPerHour == 1 && state.dataGlobal->NumOfTimeStepInHour > 1 &&
+                state.dataEnvrn->RunPeriodEnvironment) {
+                state.dataEnvrn->IsRain = state.dataWaterData->RainFall.CurrentAmount >= state.dataWeatherManager->IsRainThreshold;
+            } else {
+                state.dataEnvrn->IsRain = state.dataWeatherManager->TodayIsRain(state.dataGlobal->TimeStep, state.dataGlobal->HourOfDay);
+            }
```

Comment:
This can reduce to:
```
elseif (state.dataWaterData->RainFall.ModeID == DataWater::RainfallMode::RainSchedDesign && state.dataEnvrn->RunPeriodEnvironment){
    state.dataEnvrn->IsRain = state.dataWaterData->RainFall.CurrentAmount >= state.dataWeatherManager->IsRainThreshold;
}

Reply:
Changed the above two